examples/getting-started-session-based/01-ETL-with-NVTabular.ipynb

```diff
@@ -126,7 +126,7 @@
    "id": "cd861fcd",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.    # Apply the preprocessing and returning an NVTabular dataset
</pre><p>Better to write <code>Apply the preprocessing and return an NVTabular dataset</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>